### PR TITLE
Including Delta Lake 2.0 blog post

### PR DESCRIPTION
**Pull request by dennyglee**
This is the Delta Lake 2.0 blog post

### CODE REVIEW

File: src/blog/2022-08-02-delta-2-0-the-foundation-of-your-data-lake-is-open/index.mdx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+As part of the[ Delta Lake 2.0 release](https://github.com/delta-io/delta/releases/tag/v2.0.0), we leveraged column mapping so that dropping a column is a metadata operation. Therefore, instead of physically modifying all of the files of the underlying table to drop a column, this can be a simple modification to the Delta transaction log (i.e. a metadata operation) to reflect the column removal. Run the following SQL command to drop a column:
+
+```sql
+ALTER TABLE myDeltaTable DROP COLUMN myColumn
+```
+
+See[ documentation](https://docs.delta.io/2.0.0/delta-batch.html#drop-columns) for more details.
+
+#### Support for dynamic partition overwrites
```
Comment (tdas) [inline src/blog/2022-08-02-delta-2-0-the-foundation-of-your-data-lake-is-open/index.mdx:112]: This is a not schema operation.

Comment (dennyglee) [reply src/blog/2022-08-02-delta-2-0-the-foundation-of-your-data-lake-is-open/index.mdx:112]: Updated per feedback

File: src/blog/2022-08-02-delta-2-0-the-foundation-of-your-data-lake-is-open/index.mdx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+![CDF Delta](/cdf-delta.png)
+
+See the[ documentation](https://docs.delta.io/2.0.0/delta-change-data-feed.html) for more details.
+
+### Schema and Partition Operations: Dynamic Partition Overwrite and Drop Columns
+
+When it comes to schema and partition operations, the following new features help with data write performance.
+
+#### Support for dropping columns as a metadata operation
```
Comment (tdas) [inline src/blog/2022-08-02-delta-2-0-the-foundation-of-your-data-lake-is-open/index.mdx:98]:
```suggestion
### Support for dropping columns
```

File: src/blog/2022-08-02-delta-2-0-the-foundation-of-your-data-lake-is-open/index.mdx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+As part of the[ Delta Lake 2.0 release](https://github.com/delta-io/delta/releases/tag/v2.0.0), we leveraged column mapping so that dropping a column is a metadata operation. Therefore, instead of physically modifying all of the files of the underlying table to drop a column, this can be a simple modification to the Delta transaction log (i.e. a metadata operation) to reflect the column removal. Run the following SQL command to drop a column:
+
+```sql
+ALTER TABLE myDeltaTable DROP COLUMN myColumn
+```
+
+See[ documentation](https://docs.delta.io/2.0.0/delta-batch.html#drop-columns) for more details.
+
+#### Support for dynamic partition overwrites
```
Comment (tdas) [inline src/blog/2022-08-02-delta-2-0-the-foundation-of-your-data-lake-is-open/index.mdx:112]:
```suggestion
### Support for Dynamic Partition Overwrites
```

**Review by tdas — APPROVED** [review]

**Pull request merged**